kpatch-build/kpatch-build

```diff
@@ -854,6 +859,27 @@ if ! "$KPATCH_MODULE"; then
 	check_pipe_status create-klp-module
 fi
 
+readelf --wide --symbols $KMOD_PATH "$TEMPDIR/patch/$MODNAME.ko" 2>/dev/null | \
```

Comment:
Do we need $KMOD_PATH if we have its static API list down below?

Reply:
Ah, right, this is left over.